contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryBase2_3.sol

```diff
@@ -379,7 +379,8 @@ abstract contract AutomationRegistryBase2_3 is ConfirmedOwner {
     uint256 fallbackPrice;
     // 2nd word only read if stale
     uint96 minSpend;
-    // 3rd word only read during cancellation
+    uint8 decimals;
```

Comment:
in the function `_setBillingConfig()` can we add a require statement that checks the `decimals` value against the `decimals()` value returned by the token? This way we can be a little safer against mis-configuration.

Reply:
Good call, I like the idea.

For the record, as per [ERC20 standard](https://eips.ethereum.org/EIPS/eip-20), `This method can be used to improve usability, but interfaces and other contracts MUST NOT expect these values to be present.` And Openzeppelin used 18 and Etherscan uses 0 when the token doesn’t specify it’s decimals. 

So unfortunately I dont think we are not guaranteed to get the decimals from a given token. Accordingly, the [Openzepplin IERC20 interface](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/IERC20.sol) doesnt have decimals().

I guess we will need to rely on the human input to get the decimal places.

Reply:
I added a check to ensure the decimals is in range of [0,24], just trying to avoid human input errors. 99.9% of the tokens should be in this range. let me know if this check is necessary or too noisy.